tox.ini

```diff
@@ -0,0 +1,18 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
```

Comment:
Why does one of the commit messages claim that you made travis use tox, when the `.travis.yml` file hasn't been changed in any of the commits?